addons/product/models/product_template.py

```diff
@@ -48,6 +49,14 @@ def _read_group_categ_id(self, categories, domain, order):
         'Sales Description', translate=True,
         help="A description of the Product that you want to communicate to your customers. "
              "This description will be copied to every Sales Order, Delivery Order and Customer Invoice/Credit Note")
+    description_ecommerce = fields.Html(
+        string="eCommerce Description",
+        translate=html_translate,
+        sanitize_overridable=True,
+        sanitize_attributes=False,
+        sanitize_form=False,
+    )
```

Comment:
`description_ecommerce` / `string="eCommerce Description"` in `product` module doesn't really makes sense.
There is no concept of ecommerce in `product`.
I am not sure why do you need to share a description field between ecommerce and POS, since those are very different modules for very different use case and very different target.

But if the ecommerce PO asked for that, he knows better than me :+1: 

Still, I would rename this field and not mention eCommerce, it's weird when you are in the `pos_self_order` module to see such a field.